dev/com.ibm.ws.concurrent.persistent/resources/com/ibm/ws/concurrent/persistent/resources/CWWKCMessages.nlsprops

```diff
@@ -51,6 +51,14 @@ CWWKC1511.retry.limit.reached.failed=CWWKC1511W: Persistent executor {0} aborted
 CWWKC1511.retry.limit.reached.failed.explanation=The task has reached the limit of consecutive retries and will not be attempted again.
 CWWKC1511.retry.limit.reached.failed.useraction=If the failures or rollbacks are unexpected, determine the cause and reschedule the task. If the rollbacks and failures are intermittent, consider increasing the retry limit. 
 
+CWWKC1520.out.of.range=CWWKC1520E: Configured value {0} for {1} is not within the range of {2} to {3}.
+CWWKC1520.out.of.range.explanation=A value that is outside of the allowed range is configured for the specified property.
+CWWKC1520.out.of.range.useraction=Configure a value within the allowed range.
+
+CWWKC1521.less.than.min=CWWKC1521E: Configured value {0} for {1} cannot be less than the configured value for {2}, which is {3}.
+CWWKC1521.less.than.min.explanation=Configuring a larger retryInterval encourages other servers to retry the failed task instead of the server on which it failed.
+CWWKC1521.less.than.min.useraction=Configure a retryInterval that is larger than the missedTaskThreshold.
```

Comment:
Change
"Configure a retryInterval that is larger than the missedTaskThreshold."
To
"Configure the retryInterval property so that it is larger than the missedTaskThreshold."

Reply:
Thanks for the improved wording.  I have made the update under https://github.com/OpenLiberty/open-liberty/pull/10318/commits/4f51778b91d33d61f5cd06e199dee2387471eb4f

Reply:
Looks good. Thanks!